codecov.yml

```diff
@@ -29,3 +29,8 @@ comment:
 
 ignore:
   - "*.ignore.*"
+  - "PimsContext.cs"
```

Comment:
not sure if people agree with me on these, but to me PimsContext, PimsBaseContext and everything in the entities folder doesn't need testing.

Swagger/healthchecks are a bit more arguable.

Reply:
based on discussion, we want to ignore any frontend models, that would include legacy models as well as Concepts. Just need to make sure to only target the models, and not the mappings.

lets apply this to the frontend API models as well. also we should be ignoring mocks.